### PR TITLE
Release 7.0.1

### PR DESCRIPTION
**Pull request by treydock**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
...
+
+[Full Changelog](https://github.com/voxpupuli/puppet-logrotate/compare/v7.0.0...v7.0.1)
+
+**Fixed bugs:**
+
+- Use default systemd timer for EL9 logrotate [\#216](https://github.com/voxpupuli/puppet-logrotate/pull/216) ([treydock](https://github.com/treydock))
+
+**Closed issues:**
+
+- Add support for systemd-timer [\#214](https://github.com/voxpupuli/puppet-logrotate/issues/214)
```
Comment (bastelfreak) [inline CHANGELOG.md:17]: added skip changelog label bevause the related PR ia already in the changelog

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
...
+- logrotate 6.1.0 not compliant with stdlib9. [\#210](https://github.com/voxpupuli/puppet-logrotate/issues/210)
```
Comment (bastelfreak) [inline CHANGELOG.md:18]: added skip changelog as well. the issue does not any value to the changelog.

File: CHANGELOG.md (hunk 1 above)
Comment (treydock) [reply CHANGELOG.md]: Updated

File: CHANGELOG.md (hunk 2 above)
Comment (treydock) [reply CHANGELOG.md]: Updated

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**